### ISSUE TITLE
Handbrake Dockerfile fails because of missing GPG server pool DNS record

### ISSUE THREAD

**Issue by ksilz** [tianon/dockerfiles#121]
Hi,

Your [Handbrake Dockerfile](https://github.com/tianon/dockerfiles/blob/master/handbrake/Dockerfile) references a GPG server pool: `gpg --batch --keyserver ha.pool.sks-keyservers.net`. That pool isn't available anymore:

<img width="696" alt="image" src="https://user-images.githubusercontent.com/1061209/122909230-e2bdd900-d34c-11eb-8f94-1e80101b3e05.png">

And the DNS record isn't there anymore:
```
 % nslookup pool.sks-keyservers.net
Server:		8.8.8.8
Address:	8.8.8.8#53

** server can't find pool.sks-keyservers.net: NXDOMAIN

%
```

As a result, the build fails in that step with a `gpg: keyserver receive failed: No name`.

**Comment by tianon** [comment]
See https://github.com/docker-library/faq#openpgp--gnupg-keys-and-verification (especially https://github.com/tianon/pgp-happy-eyeballs) for why I haven't actually been affected by this directly, but yes, this should definitely be updated to a different keyserver now that SKS is dead.

**Comment by tianon** [comment]
Fixed via 04e78b5 :+1:

**Comment by ksilz** [comment]
@tianon Thank you! The `keyserver.ubuntu.com` worked like a charm.
